style.css

```diff
@@ -27,6 +27,10 @@ body {
   display: flex;
 }
 
+.no-scroll {
+  overflow-y: hidden;
+}
+
```

Comment:
- [ ] Your popup looks beautiful but there is one problem with it. Some of the descriptions for each project can not be viewed on both mobile and desktop views. I can not see the buttons that link to `See Live` and `See Source` as well. I would suggest you make all contents fit on a page or make them scrollable which I recommend (Not compulsory).

### Your Project

![image](https://user-images.githubusercontent.com/77106962/152543459-c5d656a3-2544-4972-99f9-4751d1a31ca1.png)

![image](https://user-images.githubusercontent.com/77106962/152543566-98f780e0-5b79-44b2-bc18-438e70ed759b.png)

### Figma Template
![image](https://user-images.githubusercontent.com/77106962/152544136-31678226-5881-487f-8ef2-691650f7a307.png)

![image](https://user-images.githubusercontent.com/77106962/152545134-c0e5331c-6c98-4cfb-b05f-cea020fd8622.png)